test/test_lyric.py

```diff
@@ -0,0 +1,205 @@
+    song = Song()
+    lyrics = Lyrics(song)
+    lyrics.create_line_from_string("Hello world this is a test.", 10, 50)
+    lyrics.create_line_from_string("Mixing symbols & numbers 12345, special #%$^characters, LoWeR uPPeR CaSe", 60, 100)
+    lyrics.create_line_from_string("émphasis on accénts, we also allow overlapping", 50, 110)
+    # This is an example of Chinese lyrics.
+    lyrics.create_line_from_string("这是一个中文歌词的例子。", 500, 550)
+    # This is an example of Japanese lyrics.
+    lyrics.create_line_from_string("これは日本語の歌詞の例です。", 600, 650)
+    return lyrics
+
+
+class TestLyrics(unittest.TestCase):
+    def test_create_line(self):
+        lyrics = create_lyrics()
+        line = lyrics.create_line(660)
+        self.assertEqual(len(lyrics._proto.lines), 6)
+        self.assertEqual(line.is_empty(), True)
+        assert_lyric_lines_equal({"sentence": "", "start_tick": 660, "end_tick": 1140}, line)
+
+    def test_create_line_from_string(self):
+        lyrics = create_lyrics()
+        line = lyrics.create_line_from_string("Hello world! 你好世界！", 10, 50)
+        self.assertEqual(len(lyrics._proto.lines), 6)
+        assert_lyric_lines_equal({
+            "sentence": "Hello world! 你好世界！",
+            "start_tick": 10,
+            "end_tick": 50
+        }, line)
+
+    def test_remove_line_at_index(self):
+        lyrics = create_lyrics()
+        lyrics.create_line(11)
+        lyrics.create_line(20)
+        self.assertEqual(len(lyrics._proto.lines), 7)
+        lyrics.remove_line_at_index(5)
+        self.assertEqual(len(lyrics._proto.lines), 6)
+        lyrics.remove_line_at_index(4)
+        lyrics.remove_line_at_index(1)
+        lyrics.remove_line_at_index(2)
+        lyrics.remove_line_at_index(1)
+        self.assertEqual(len(lyrics._proto.lines), 2)
+        assert_lyric_lines_equal({
+            "sentence": "Hello world this is a test.",
+            "start_tick": 10,
+            "end_tick": 50
+        }, lyrics.get_line_at_index(0))
+        
+    def test_clone_line(self):
+        lyrics = create_lyrics()
+        line = lyrics.create_line(300)
+        new_line = lyrics.clone_line(line)
+        self.assertEqual(len(lyrics._proto.lines), 7)
+        # Check the cloned line
+        assert_lyric_lines_equal({
+            "sentence": "",
+            "start_tick": 300,
+            "end_tick": 780
+        }, new_line)
+        # Check if cloned and sorted properly
+        assert_lyric_lines_equal({
+            "sentence": "",
+            "start_tick": 300,
+            "end_tick": 780
+        }, lyrics.get_line_at_index(3))
+        assert_lyric_lines_equal({
+            "sentence": "",
+            "start_tick": 300,
+            "end_tick": 780
+        }, lyrics.get_line_at_index(4))
+
+    def test_clear(self):
+        lyrics = create_lyrics()
+        lyrics.create_line(10)
+        lyrics.clear()
+        self.assertEqual(len(lyrics._proto.lines), 0)
+
+
+class TestLyricLine(unittest.TestCase):
```

Comment:
You also need to provide a method to update the line with a string and test it here.